config/configuration.toml.in

```diff
@@ -114,3 +114,19 @@ path = "@SHIMPATH@"
 # system log
 # (default: disabled)
 #enable_debug = true
+#
+# Internetworking model
+# Determines how the VM should be connected to the
+# the container network interface
+# Options:
+#
+#   - bridged
+#     Uses a linux bridge to interconnect
+#     the container interface to the VM. This is the
+#     safe default that works for most cases except
```

Comment:
I know exactly what you mean, but I'd remove the word "safe" here as it could imply the other options are unsafe ;) You could just say:

> Works for most cases except macvlan and ipvlan.